openstudiocore/src/model/EnergyManagementSystemActuator.hpp

```diff
@@ -0,0 +1,109 @@
+/**********************************************************************
+ *  Copyright (c) 2008-2016, Alliance for Sustainable Energy.
+ *  All rights reserved.
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  License along with this library; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ **********************************************************************/
+
+#ifndef MODEL_ENERGYMANAGEMENTSYSTEMACTUATOR_HPP
+#define MODEL_ENERGYMANAGEMENTSYSTEMACTUATOR_HPP
+
+#include <model/ModelAPI.hpp>
+#include "ModelObject.hpp"
+
+namespace openstudio {
+
+namespace energyplus { 
+  class ReverseTranslator;
```

Comment:
I made the "incomplete" constructors private so that a user could not, but the ReverseTranslator could create an incomplete EMS object (by incomplete i mean somethings thats missing a required field).  This covers the case where a pointed to object doesnt RT, so we make the EMS object as best we can and then try and give comments as to what the untranslated object was.  This way a user can play with RT an older EMS program to see what it would look like with the new stuff.